airbyte-integrations/connectors/source-okta/source_okta/source.py

```diff
@@ -86,7 +86,7 @@ def cursor_field(self) -> str:
         pass
 
     def get_updated_state(self, current_stream_state: MutableMapping[str, Any], latest_record: Mapping[str, Any]) -> Mapping[str, Any]:
-        min_cursor_value = self.min_id if self.min_id else str(pendulum.datetime.min)
```

Comment:
Looks like this change might be causing `TestIncremental.test_two_sequential_reads[inputs0]` to fail:

```
>           assert compare_cursor_with_threshold(
                record_value, state_value, threshold_days
            ), f"Second incremental sync should produce records older or equal to cursor value from the state. Stream: ***stream_name***"
E           AssertionError: Second incremental sync should produce records older or equal to cursor value from the state. Stream: group_members
``` 
The change is incorrectly updating the cursor/state.

In the test logs:
https://github.com/airbytehq/airbyte/actions/runs/3982519582/jobs/6827064620#step:12:7646

Reply:
And it might also be causing `TestIncremental.test_read_sequential_slices[inputs0]` to fail:

```
>               assert compare_cursor_with_threshold(
                    record_value, state_value, threshold_days
                ), f"Second incremental sync should produce records older or equal to cursor value from the state. Stream: ***stream_name***"
E               AssertionError: Second incremental sync should produce records older or equal to cursor value from the state. Stream: group_members
```
Link to test run:
https://github.com/airbytehq/airbyte/actions/runs/3982519582/jobs/6827064620#step:12:7719

Reply:
In `TestIncremental.test_state_with_abnormally_large_values[inputs0]` the state is also causing errors:

```
>       assert (
            not records
        ), f"The sync should produce no records when run with the state with abnormally large values ***records[0].record.stream***"
E       AssertionError: The sync should produce no records when run with the state with abnormally large values group_members
```
Link to test run:

https://github.com/airbytehq/airbyte/actions/runs/3982519582/jobs/6827064620#step:12:7741

Reply:
Ohhh. I see, thank you so much @natalyjazzviolin ! I had a problem on this line when I mistakenly installed the latest version of the library. Let me quickly revert the changes for this.

Reply:
Done reverting the changes made for min_cursor_value.